network_simulator/flow.py

```diff
@@ -275,15 +773,28 @@ class FlowTimeoutPacketEvent(Event):
     def __init__(self, flow, packet):
         """
         :ivar Flow flow: flow of this event.
-        :ivar packet packet: the particular packet of this timeout.
+        :ivar Packet packet: the particular packet of this timeout.
+        :ivar int old_pending_timeouts: The number of timeouts that were
+        pending for this Packet during the run() call. This is used to check
+        whether we need to handle packet losses.
         """
         super(FlowTimeoutPacketEvent, self).__init__()
         self.flow = flow
         self.packet = packet
+        self.old_pending_timeouts = None
```

Comment:
Again it doesn't matter because it's set immediately in run(), and does not make sense to set before run() runs (since it tracks the old number right before run() was called).